bin/cron/analyze_hoc_activity

```diff
@@ -51,10 +51,10 @@ def main
   # https://docs.google.com/document/d/1RTTCpkDYZjqZxfVehkZRkk1HckYMvFdFGs6SEZnK1I8
   total_started += 14_861_327
 
-  today = DateTime.now.to_date
-  day = Date.strptime('2014/12/06', '%Y/%m/%d')
+  today = Date.strptime('2020/02/01', '%Y/%m/%d')
+  day = Date.strptime('2020/02/01', '%Y/%m/%d')
 
-  while day <= today
+  while day < today
```

Comment:
`day` and `today` are equal, right (Feb 1, 2020)?  Will this entire `while` loop be skipped?

Reply:
True -- I changed this because (please confirm my logic) the `analyze_hoc_activity` cronjob was scheduled to run every hour on the :35, so what ended up actually happening is that the counts for a given day would be run (and overwritten every hour) until 11:35 PM, which would represent the "entire day" of data. This (I think) will result in only running this script when we have a full day's data (I'm imagining if we turn it back on, we should switch the cron schedule just to run daily, instead of hourly).

Reply:
So, like you said, this actually won't run as-is, but my thought was that for running this manually, we can just modify the dates to whatever we'd like once it's on `production-daemon`.

Reply:
OK - now I understand.  Looks great!  Nice that you were able to run this on the clone. :)